### PR TITLE
#1513

### PR DESCRIPTION
**Pull request by NikolaiZheleznyi**

### CODE REVIEW

File: carina-api/src/main/java/com/qaprosoft/apitools/builder/MessageBuilder.java [hunk 1]
```diff
@@ -23,9 +23,12 @@
 import freemarker.template.Configuration;
 import freemarker.template.Template;
 import freemarker.template.TemplateException;
+import org.apache.log4j.Logger;
 
 public class MessageBuilder {
 
+    private static final Logger LOGGER = Logger.getLogger(MessageBuilder.class);
```
Comment (vdelendik) [inline carina-api/src/main/java/com/qaprosoft/apitools/builder/MessageBuilder.java:30]: still log4j directly? that's valid declaration of logger:
` private static final Logger LOGGER = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());`

File: carina-core/pom.xml [hunk 2]
```diff
@@ -96,9 +98,10 @@
 
 		<!-- Test utilities -->
 		<dependency>
-			<groupId>org.testng</groupId>
```
Comment (vdelendik) [inline carina-core/pom.xml:99]: removal of testng is incorrect. we can't reuse transitive dependency here and must use the version declared on our end! Please return!

File: pom.xml [hunk 3]
```diff
@@ -43,6 +43,7 @@
         <maven-javadoc-plugin.version>3.0.1</maven-javadoc-plugin.version>
         <maven-gpg-plugin.version>1.6</maven-gpg-plugin.version>
         <jacoco-maven-plugin>0.8.7</jacoco-maven-plugin>
+        <zebrunner-agent.version>RELEASE</zebrunner-agent.version>
```
Comment (vdelendik) [inline pom.xml:46]: specify concrete version here: 1.6.6

File: pom.xml [hunk 4]
```diff
@@ -647,12 +649,13 @@
...
-  <groupId>com.zebrunner</groupId>
-  <artifactId>agent-testng</artifactId>
-  <version>RELEASE</version>
-  <scope>test</scope>
-</dependency>
+
+            <dependency>
+              <groupId>com.zebrunner</groupId>
+              <artifactId>agent-testng</artifactId>
+              <version>RELEASE</version>
```
Comment (vdelendik) [inline pom.xml:656]: reuse `zebrunner-agent.version` maven properties you declared at the top

File: log4j.properties [hunk 5]
```diff
@@ -0,0 +1,4 @@
+log4j.rootLogger = INFO, zebrunner
```
Comment (vdelendik) [inline log4j.properties:1]: why do we need this file here?

File: carina-core/pom.xml (hunk 2 above)
Comment (NikolaiZheleznyi) [reply carina-core/pom.xml:99]: done

File: pom.xml (hunk 3 above)
Comment (NikolaiZheleznyi) [reply pom.xml]: done

File: pom.xml (hunk 4 above)
Comment (NikolaiZheleznyi) [reply pom.xml]: done

File: log4j.properties (hunk 5 above)
Comment (NikolaiZheleznyi) [reply log4j.properties]: done, file deleted

**Pull request merged**